analyzer/src/funTest/assets/projects/synthetic/spdx/project-xyz-with-sub-project-dependencies.spdx.yml

```diff
@@ -0,0 +1,43 @@
+SPDXID: "SPDXRef-DOCUMENT"
```

Comment:
commit-message: How is that "referencing" between projects relevant to this commit? Could you amend the commit message to explain the intended behavior?

(Is it so that if a SPDXDocument is referenced, then it has to be a package?)

Reply:
I have reworded the commit message, no longer pointing out a specific reason for the problem, but also stating that there might be changes in the heuristic to detect packages or projects.